File: src/test/java/ac/dnd/server/annotation/UnitTest.java
package ac.dnd.server.annotation;

import java.lang.annotation.ElementType;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;
import java.lang.annotation.Target;

import org.junit.jupiter.api.DisplayNameGeneration;
import org.junit.jupiter.api.DisplayNameGenerator;
import org.junit.jupiter.api.Tag;

@Target(value = ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
@Tag("UnitTest")
public @interface UnitTest {
}

File: src/test/kotlin/ac/dnd/server/annotation/UnitTest.kt
package ac.dnd.server.annotation

import org.junit.jupiter.api.DisplayNameGeneration
import org.junit.jupiter.api.DisplayNameGenerator
import org.junit.jupiter.api.Tag

@Target(AnnotationTarget.CLASS)
@Retention(AnnotationRetention.RUNTIME)
@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores::class)
@Tag("UnitTest")
annotation class UnitTest